js/script.js

```diff
@@ -1,57 +1,123 @@
+const bookList = document.getElementById('book-list');
+const addBtn = document.getElementById('add-button');
+const bookTitleInput = document.getElementById('title');
+const bookAuthorInput = document.getElementById('author');
+const alertMessage = document.getElementById('alert-message');
+
 let books = [];
 
-const renderBooks = (books) => {
-  const addBook = (title, author) => {
-    const bookList = document.getElementById('book-list');
-    const book = document.createElement('div');
-    const bookTitle = document.createElement('p');
-    bookTitle.textContent = `Title: ${title}`;
-    const bookAuthor = document.createElement('p');
-    bookAuthor.textContent = `Author: ${author}`;
-    const removeBtn = document.createElement('button');
-    removeBtn.textContent = 'Remove';
-
-    const deleteBook = (bookTitle) => {
-      const result = books.filter((book) => book.title !== bookTitle);
-      books = result;
-      localStorage.setItem('books', JSON.stringify(books));
-      renderBooks(books);
-    };
-
-    removeBtn.addEventListener('click', () => {
-      deleteBook(title);
-    });
-
-    const bookHr = document.createElement('hr');
-
-    book.append(bookTitle, bookAuthor, removeBtn, bookHr);
-    bookList.append(book);
-  };
-
-  const bookList = document.getElementById('book-list');
-  bookList.innerHTML = '';
+function bookBackground(book, element) {
+  if (element % 2 === 0) {
+    book.style.backgroundColor = 'rgb(214, 214, 214)';
+  } else {
+    book.style.backgroundColor = 'azure';
+  }
+}
```

Comment:
- [ ] [OPTIONAL] This file has too many lines at the moment. You can extract some related functionalities to separate files to make your code clean and modular